autoload/maktaba.vim

```diff
@@ -105,3 +110,22 @@ function! maktaba#LateLoad(...) abort
     call maktaba#filetype#Cycle()
   endif
 endfunction
+
+
+""
+" @private
+" Forces the disabling of the Python implementation of the maktaba#json
+" functions, to enable testing. This must be called before referencing any of
+" the maktaba#json functions, since it will only take effect on first load.
+function! maktaba#SetJsonPythonDisabled(disabled) abort
```

Comment:
There's no way we can make it work at `maktaba#json#SetPythonDisabled` and still be able to trigger it early enough in init, is there? Possibly w/ some changes in maktaba.vim to defer some initialization?